### DISCUSSION TITLE
How to get binaries

### DISCUSSION THREAD

**Discussion by HengYeDev** [perpetualKid/FreeTrainSimulator#52] [Q&A] [answered]
The updater is broken on Linux but I can run the binaries. Is there a way to get the nightly build .exe files?

**Comment by perpetualKid** [comment, marked as answer]
The news headers in the wiki section contain links to the relevant versions at that point, i.e.
https://ultimaterails.blob.core.windows.net/builds/1.3.6/ci/1.3.6-ci.16.zip

You can follow the pattern, and for the current build **1.3.7-ci.7** this would be [https://ultimaterails.blob.core.windows.net/builds/**1.3.7/ci/1.3.7-ci.7**.zip](https://ultimaterails.blob.core.windows.net/builds/1.3.7/ci/1.3.7-ci.7.zip)

There currently is no separately published list of builds, but you could follow the build status on the this repos [home page] (https://github.com/perpetualKid/ORTS-MG/blob/Monogame/README.md) towards the bottom
![image](https://user-images.githubusercontent.com/7158903/112332192-d11b5580-8cb9-11eb-9554-f765f68f54ad.png)
and click on the **Continuous Integration Builds** --> [![Build Status](https://dev.azure.com/perpetualKid/ORTS-MG/_apis/build/status/Build/CI%20Builds)](https://dev.azure.com/perpetualKid/ORTS-MG/_build?definitionId=11). This will link to the Azure DevOps build history which reveals the build name (version)
